sql/core/src/test/scala/org/apache/spark/sql/TestData.scala

```diff
@@ -203,4 +203,17 @@ object TestData {
         :: ComplexData(Map("2" -> 2), TestData(2, "2"), Seq(2), false)
         :: Nil).toDF()
   complexData.registerTempTable("complexData")
+
+  case class OrderByData(a: String, b: Int)
+  val orderByData =
+    TestSQLContext.sparkContext.parallelize(
+      OrderByData("1", 3) ::
+        OrderByData("1", 4) ::
+        OrderByData("2", 7) ::
+        OrderByData("2", 8) ::
+        OrderByData("3", 5) ::
+        OrderByData("3", 6) ::
+        OrderByData("4", 1) ::
+        OrderByData("4", 2) :: Nil).toDF()
+  orderByData.registerTempTable("orderByData")
```

Comment:
Since this `orderByData` is only used in one test case, you should just create this data frame in that test case, not make it global here.